jobs/integration/test_autoscaler.py

```diff
@@ -29,6 +29,7 @@ async def deployment(kubectl, namespace):
     kubectl.create.namespace(namespace)
     kubectl.create(f=path_to_deployment, namespace=namespace)
     yield path_to_deployment
+    kubectl.delete(f=path_to_deployment, namespace=namespace, grace_period=5 * 60)
```

Comment:
why wait 5m vs the default 30s here?

Reply:
maybe this is the wrong timeout value.  Currently the following command (the one to delete the namespace) never completes.  It just sits there forever.  I was hoping by adding a grace-period, it would quit after 5m.  Maybe i'm reading this doc wrong